### PR TITLE
[RFC] bot,stale: add stale plugin config

### PR DESCRIPTION
**Pull request by champtar**
We have a lot of stale issues and PR
Let's automatically close them so we can concentrate on the active ones
https://probot.github.io/apps/stale/

### CODE REVIEW

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 45
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 15
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - security
```
Comment (karlp) [inline .github/stale.yml]: if it's gone 45 days with no action, it's not really a "security" issue is it? :)

File: .github/stale.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 45
```
Comment (karlp) [inline .github/stale.yml]: I'd rather see this more like 90 and 30, but that's really just bikeshedding I guess.

Comment (champtar) [reply .github/stale.yml]: my idea is 2 fold:
1) send a remainder that this issue / PR need actions, so the sooner the better
2) when it's really stale close it
Closing is not a death sentence, we can reopen
Also a new comment restart the stale counter, if nobody comment for 60 days I think we can call that stale

**Comment by thess** [comment]
Any time-frame is better than none.
Good idea - let's do it.

File: .github/stale.yml (hunk 2 above)
Comment (karlp) [reply .github/stale.yml]: I'm sold :)

**Comment by diizzyy** [comment]
In all honestly, the majority of open PRs are either:

1. Adding new software and no one with commit access wants to actually decide whether to add it or not 2. PRs without any run-tests

I don't think we're doing anyone favors especially closing PRs to hide the underlying issues, it also discourages people to contribute if they don't receive a human reply with a reasonable argument.

I personally don't like the mass PRs without any run tests (or at least mentioned ones). These should not be submitted in the first place but that's just my personal opinion as it more or less just piles up.

Closing because of submitter timeout is fine but that only applies to a very small amount of open PRs.

**Comment by dibdot** [comment]
@champtar I find it very useful, especially for issue reporting that lacks bare minimum information like maintainer, logs etc. Thanks!

**Comment by diizzyy** [comment]
I guess we can close this now?

**Comment by diizzyy** [comment]
...I guess so

**Pull request closed without merging**